src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double longerSide;
+    private final double height;
```

Comment:
In the class Rectangle, the naming of the variable 'height' can cause misunderstanding. The height of the rectangle is commonly referred as the 'shorter side'. Consider renaming it to 'shorterSide' or 'width'.

Reply:
done